_dev/.stylelintrc

```diff
@@ -22,6 +22,7 @@
           "each",
           "include",
           "mixin",
+          "use",
```

Comment:
I had to add this rule in here as we are using `@use` in our scss files and the pre-commit hooks were failing